src/Sql/Sql/ManagedInstance/Cmdlet/SetAzureSqlManagedInstance.cs

```diff
@@ -130,6 +130,25 @@ public class SetAzureSqlManagedInstance : ManagedInstanceCmdletBase
             HelpMessage = "Determines how much VCore to associate with instance")]
         public int? VCore { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not the public data endpoint is enabled.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Whether or not the public data endpoint is enabled for the instance.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter(Constants.PublicDataEndpointEnabledTrue, Constants.PublicDataEndpointEnabledFalse)]
```

Comment:
I don't think argument completer is needed for boolean

Reply:
I can remove it if it's needed.

Reply:
Can you please confirm that I should remove this boolean argument completer.

Reply:
I have never seen argument completer specified for boolean. PowerShell team would be able to give better guidance than me on this one.